TheWidlarzGroup/react-native-video
# fix(android): ads build and enable ads in android sample

Ensure Sample builds correctly with IMA enable
Related issue: https://github.com/react-native-video/react-native-video/issues/3370

- Fix bad ima import
- Fix missing rework in ima stub
- Apply kotlin workaround
- Apply androidx.core workaround
- enable RNVUseExoplayerIMA = true in sample by default

### how to test
go to sample directory
yarn android
--> you should be able to see ads on big bug bunny sample
comment RNVUseExoplayerIMA = true
yarn android
--> you should NOT be able to see ads on big bug bunny sample (but playback works anyway !)
